### PR TITLE
feat: add delegates page

### PR DESCRIPTION
**Pull request by Sekhmet**
## Summary

Closes: https://github.com/snapshot-labs/sx-ui/issues/566

This PR adds adds delegates page that can be added to the space by configuring API url.

## Test plan

- Create new space or edit existing space profile to add delegation API URL (for example `https://api.thegraph.com/subgraphs/name/arr00/uniswap-governance-v2`).
- Go to space page.
- Go to Delegates page.

## Screenshots

image

### CODE REVIEW

**Comment by bonustrack** [comment]
Looks great! Can you just:
- Remove the "delegator(s)" column on mobile
- Add an input for the settings where you can select the type / standard being used for the API, this will help later once we deal with others standard, for Compound based subgraph it can be "Governor subgraph" id : `governor-subgraph`

File: src/components/Modal/EditSpace.vue [hunk 1]
```diff
@@ -12,7 +12,8 @@ const DEFAULT_FORM_STATE: SpaceMetadata = {
   discord: '',
   votingPowerSymbol: '',
   walletNetwork: null,
-  walletAddress: null
+  walletAddress: null,
+  delegationApiUrl: ''
```
Comment (samuveth) [inline src/components/Modal/EditSpace.vue:16]: Shouldn't this be an array, or how do you imagine supporting multiple networks? @bonustrack

Comment (samuveth) [reply src/components/Modal/EditSpace.vue:16]: I imagine something like this for Snapshot: https://github.com/orgs/snapshot-labs/projects/11/views/7?pane=issue&itemId=27520397

Comment (bonustrack) [reply src/components/Modal/EditSpace.vue:16]: If a space has a delegation system on multiple network he would need an unified API that merge voting power, we shouldnt support multiple network here.

Comment (samuveth) [reply src/components/Modal/EditSpace.vue:16]: Why are we not doing that also for our own delegation subgraphs? Instead we have all these:

```json
{
  "1": "https://gateway.thegraph.com/api/0f15b42bdeff7a063a4e1757d7e2f99e/deployments/id/QmXZiV6S13ha6QXq4dmaM3TB4CHcDxBMvGexSNu9Kc28EH",
  "5": "https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-goerli",
  "10": "https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-optimism",
  "56": "https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-binance-smart-chain",
  "100": "https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-gnosis-chain",
  "137": "https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-polygon",
  "250": "https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-fantom",
  "42161": "https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-arbitrum"
}
```

Comment (samuveth) [reply src/components/Modal/EditSpace.vue:16]: This is also very slow when loading on the Ui for some reason

Comment (bonustrack) [reply src/components/Modal/EditSpace.vue:16]: Our subgraph have only the mapping of delegation not the voting power, this is why its super slow

Comment (samuveth) [reply src/components/Modal/EditSpace.vue:16]: > If a space has a delegation system on multiple network he would need an unified API that merge voting power, we shouldnt support multiple network here.

But should be support multiple URLs? Quote from the Delegation pitch "providing links to the relevant APIs", this does sound like it should support multiple URLs

Comment (samuveth) [reply src/components/Modal/EditSpace.vue:16]: Once we have multiple standards, it would also be beneficial to consider supporting them simultaneously.

Comment (samuveth) [reply src/components/Modal/EditSpace.vue:16]: If a user clicks delegate on the Ui, how does the Ui know on which network the delegation should happen by the way?

**Pull request merged**